README.md

```diff
@@ -225,8 +225,8 @@ end
 Route patterns may have optional parameters:
 
 ``` ruby
-get '/posts.?:format?' do
-  # matches "GET /posts" and any extension "GET /posts.json", "GET /posts.xml" etc.
+get '/post/:format?' do
+  # matches "GET /posts" and any extension "GET /posts/json", "GET /posts/xml" etc
```

Comment:
I feel the description quite doesn't fit here since it looks like a path fragment rather than an extension.  Perhaps reword this?